Src/CZICmd/cmdlineoptions.cpp

```diff
@@ -34,19 +34,19 @@ struct PlaneCoordinateValidator : public CLI::Validator
     {
         this->name_ = "PlaneCoordinate";
         this->func_ = [](const std::string& str) -> string
-        {
-            try
-            {
-                auto plane_coordinate = libCZI::CDimCoordinate::Parse(str.c_str());
-                return {}; // returning "" means "validation was successful" (c.f. https://cliutils.github.io/CLI11/book/chapters/validators.html)
-            }
-            catch (libCZI::LibCZIStringParseException&)
             {
```

Comment:
Are the changes in this file only due to automatic reformatting? (Any change in the formatting rules?)

Reply:
It also appears just like formatting changes to me for this block

Reply:
Yeah, those "formatting changes" are pesky. Not sure why this is - or, one reason I am aware of: it seems that in recent VisualStudio-version some bugs in the formatting-engine have been fixed (I remember that with lambdas, there have been issues, which with latest version seem gone). So, not sure what to do here - maybe the .editorconfig-file needs some improvements? However, for the time being - I guess we have to live with it.

Reply:
...but, sure, there are functional changes in this file:
- adding two new switches '--source-stream-class' and '--propbag-source-stream-creation'
- listing the available stream-class with the switch '--version'
![image](https://github.com/ZEISS/libczi/assets/4881321/52874781-f05c-4928-a8bc-0b936479d2ae)